src/openms/source/CHEMISTRY/EnzymaticDigestionLogModel.cpp

```diff
@@ -175,8 +176,9 @@ namespace OpenMS
     output.clear();
     AASequence::ConstIterator begin = protein.begin();
     AASequence::ConstIterator end = protein.begin();
-    while (nextCleavageSite_(protein, end), end != protein.end())
+    while (end != protein.end())
     {
+      nextCleavageSite_(protein, end)
```

Comment:
Is this correct?

Reply:
if it was correct before then it now produces a different result.
Note that 
while (nextCleavageSite_(protein, end), end != protein.end()) {}
is exactly the same as
nextCleavageSite_(protein, end);
while (end != protein.end()) {}